src/jobsautoreport/main.py

```diff
@@ -27,35 +29,30 @@ def main() -> None:
     querier = Querier(client, index_name)
     reporter = Reporter(querier)
 
+    data = {}
```

Comment:
```suggestion
    data = {
      "success_rate": reporter.get_success_rate(a_week_ago, now),
      "number_of_jobs_triggered": str(reporter.get_number_of_jobs_triggered(a_week_ago, now)),
      "average_jobs_duration": reporter.get_average_duration_of_jobs(a_week_ago, now),
    }
```
What does it mean when duration is none? Can we be more explicit about it? Maybe throw (and catch) an exception?